addons/hr_holidays/static/src/leave_stats/leave_stats.xml

```diff
@@ -1,30 +1,48 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <templates xml:space="preserve">
     <div t-name="hr_holidays.LeaveStatsComponent" class="o_leave_stats">
-        <div t-if="employee" id="o_leave_stats_employee">
-            <div class="o_hr_leave_subtitle">
-                <t t-esc="employee[1]"/> in <t t-esc="thisYear"/>
+        <t t-if="state.multi_employee" id="o_leave_stats_multi_employee">
```

Comment:
We have `Information for multiple employees cannot be displayed` three times. Instead, what do you think if we do something like this?

```
<t t-if="state.multi_employee or state.holiday_type in ['company',  'category']" id="o_leave_stats_multi_employee">
    <div class="fst-italic">
        Information for multiple employees cannot be displayed
    </div>
<t t-elif="state.employee" id="o_leave_stats_employee">
    xxxx
</t>
<t t-elif="state.department" id="o_leave_stats_department">
    xxxx
</t>
```